examples/react/tutorial/step1-login/README.md

```diff
@@ -1,18 +1,46 @@
+# Getting Started with Create React App
```

Comment:
Please update the readme to reflect what we are trying to accomplish with this example.

Reply:
Done!
I copied the readme that was before!